coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 97.05
+    "covered_percent": 72.14
```

Comment:
:(

Reply:
This doesn't seem correct. The majority of added code here was in views, which don't factor into test coverage. Do you have test failures or was this perhaps added after running an incomplete test suite?

Reply:
@Chauce can we re-rerun the specs to verify this percentage?